### DISCUSSION TITLE
Add milestones

### DISCUSSION THREAD

**Discussion by nils-a** [community/community#15192] [Mobile]
I often add labels and milestones to PRs using the mobile app.
While labels are mostly set up, I often encounter the problem that no milestone has been set up.

So, what I am missing is the ability to create a new milestone in the mobile app.

Ideally, the whole flow of creating, viewing and modifying milestones should be available. But I'll settle for "create".

Thanks for the wonderful app. 🎉

**Comment by haldun** [comment]
Thank you for the feedback. I have created a tracking issue for this and passing it to the team to do prioritization.

**Reply by DanielBaulig** [reply]
It seems like this was never prioritized. Any chance we can get this feature? It seems like a very low hanging fruit that would make our lives for us app users so much easier.

**Reply by GLD5000** [reply]
Agreed- this would be really useful!

**Reply by LeonhardEulerr** [reply]
Any update on that?

**Reply by djcavei** [reply]
Any update?

**Reply by jrlanglois** [reply]
Any updates on this?

**Reply by Michael-L** [reply]
Any updates? This could be a quick win for a somewhat interesting feature.

**Reply by vananiev** [reply]
Any update?

**Reply by baalIsBack** [reply]
Would love this feature

**Reply by lukasJambit** [reply]
Any update on this topic?

**Reply by Tuevon** [reply]
Any update?

**Reply by reetibee** [reply]
Any update?

**Reply by jrlanglois** [reply]
This feature is basically a YOLO code able thing that AI could probably do ok in getting a leg up quickly... which is better than silence and 0 changes in this regard.

**Comment by marjorieazoulay04-cyber** [comment]
> I often add labels and milestones to PRs using the mobile app.
> While labels are mostly set up, I often encounter the problem that no milestone has been set up.
> 
> So, what I am missing is the ability to create a new milestone in the mobile app.
> 
> Ideally, the whole flow of creating, viewing and modifying milestones should be available. But I'll settle for "create".
> 
> Thanks for the wonderful app. 🎉

[a comment by an automated account is not shown]
